### PR TITLE
Classes vs struct

### PR DESCRIPTION
**Pull request by Manithj**

### CODE REVIEW

File: Documentation/3. Classes vs Struct/README.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+ }
+ ```
+
+ C++ object class
+
+ ```c++
+#include <iostream>  
+using namespace std;  
+   class Student {  
+      public:
```
Comment (PramudithaPothuwila) [inline Documentation/3. Classes vs Struct/README.md]: Remove Indentation of public:

File: Documentation/3. Classes vs Struct/README.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+### <u>C++ Class</u>
+
+In C++, class is a group of similar objects. It is a template from which objects are created. It can have fields, methods, constructors etc.
+
+eg:-
+```c++
+class Student    
+ {    
+     public:
```
Comment (PramudithaPothuwila) [inline Documentation/3. Classes vs Struct/README.md]: Remove Indentation

**Review by PramudithaPothuwila — CHANGES_REQUESTED** [review]
Remove Indentation mentioned in the comment

File: Documentation/3. Classes vs Struct/README.md (hunk 1 above)
Comment (Manithj) [reply Documentation/3. Classes vs Struct/README.md]: Done

File: Documentation/3. Classes vs Struct/README.md (hunk 2 above)
Comment (Manithj) [reply Documentation/3. Classes vs Struct/README.md]: Done

File: Documentation/3. Classes vs Struct/README.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+## <u>Classes vs Struct</u>
+
+### <u>C++ Class</u>
+
+In C++, class is a group of similar objects. It is a template from which objects are created. It can have fields, methods, constructors etc.
+
+eg:-
+```c++
+class Student    
+{
```
Comment (PramudithaPothuwila) [inline Documentation/3. Classes vs Struct/README.md:10]:
```C++
class Student    
{    
public:  
   int id;  //field or data member     
   float salary; //field or data member  
   String name;//field or data member    
}
```

**Review by PramudithaPothuwila — CHANGES_REQUESTED** [review]

File: Documentation/3. Classes vs Struct/README.md (hunk 3 above)
Comment (Manithj) [reply Documentation/3. Classes vs Struct/README.md:10]: fixed

**Review by PramudithaPothuwila — APPROVED** [review]

**Pull request merged**